readme.md

```diff
@@ -137,26 +181,43 @@ $ npm -v
 
 * **Your Answer:**
 
----
+Hello, Node!
+Toan Bui
+C:\source\Summer2019\w1-node-ecosystem
+
 
 - [ ] Install the [moment](https://www.npmjs.com/package/moment) package
 
 * **Question:** What command can you run to install this package?
 
 * **Your Answer:**
 
----
+npm install moment --save
+https://momentjs.com/
+
 
 - [ ] On your own, use this package in the `index.js` file
 
 * **Question:** Do you need to use a `./` to require the package? Why or why not?
 
 * **Your Answer:**
 
----
+const moment = require('./node_modules/moment/moment.js')
```

Comment:
While you can do this, you do not need to. You only need to do the following:
```js
const moment = require('moment')
```
This will find the package in the `node_modules/` folder by default.